src/inspector_socket_server.cc

```diff
@@ -376,11 +385,25 @@ void InspectorSocketServer::SendListResponse(InspectorSocket* socket) {
 
 bool InspectorSocketServer::Start() {
   CHECK_EQ(state_, ServerState::kNew);
-  sockaddr_in addr;
+  struct addrinfo hints;
+  memset(&hints, 0, sizeof(struct addrinfo));
```

Comment:
In my opinion, made the code slightly easier - prefills the sockaddr structures.